### ISSUE TITLE
Use 'Album artist' instead of 'Artist' when grouping albums

### ISSUE THREAD

**Issue by rockertux** [flaviotordini/musique#27]
Whenever an album has multiple `Artist`s tags in its songs, Musique treats the album as separate records. For example, metatag imported from [this MusicBrainz entry](https://musicbrainz.org/release/c596f278-4e9d-3fbb-8db0-175c73bb5a22) generates two "Keeper of the Seven Keys: The Legacy" in the album listing, one with songs tagged "Helloween" and one with the song tagged "Helloween feat. Candice Night". There is, though, a tag named `Album artist` where all songs are tagged "Helloween". MusicBrainz also has some albums with only the `Album artist` tag set, for example [this one](https://musicbrainz.org/release/48117456-5b07-43e7-b881-764c87a24a06), which makes the album not being listed under the artist albums in Musique. I think it'd be smarter to prioritize `Album artist` and fall back to `Artist` when that tag cannot be found. 😄

**Comment by rockertux** [comment]
On a related note, I tried building it to fix it myself and there's a missing header for the MacOS.

**Comment by flaviotordini** [comment]
Yes, this is a long standing issue. I'll look into it for a major release.
